source/tyr/tyr/resources.py

```diff
@@ -2073,17 +2073,19 @@ def _get_all_backends(self):
 
         if pagination.has_next:
             pagination_json['next'] = url_for(request.endpoint, page=pagination.next_num)
-        return marshal(pagination.items, streetnetwork_backend_fields), pagination_json
+        return pagination.items, pagination_json
```

Comment:
Can't you return the pagination item instead of a tuple? It should be easier for the eye to process it afterwards.